vent/coordinator/process_manager.py

```diff
@@ -0,0 +1,39 @@
+import threading
+import time
+
+
+class ProcessManager:
+    # Functions:
+    def __init__(self, startCommandLine=None, maxHeartbeatInterval=None):
+        self.thread = threading.Thread(target=self.start)
+        self.thread.start()
+        self.command_line = None  # TODO: what is this?
```

Comment:
This is the command the ProcessManager would issue to start the ControlModule process.